src/scenekit.cs

```diff
@@ -3164,13 +3164,6 @@ interface SCNSceneRenderer {
 		[DebuggerBrowsable (DebuggerBrowsableState.Never)]
 		SCNNode AudioListener { get; set; }
 
-#if XAMCORE_4_0
-		[Abstract] // this protocol existed before iOS 9 (or OSX 10.11) and we cannot add abstract members to it (breaking changes)
-#endif
-		[Watch (6,0), TV (13,0), Mac (10,15), iOS (13,0)]
-		[Export ("temporalAntialiasingEnabled")]
```

Comment:
This seems to be quite public: https://developer.apple.com/documentation/scenekit/scnscenerenderer/3227912-temporalantialiasingenabled?changes=l_8&language=objc